### ISSUE TITLE
Latest version prints older one to terminal

### ISSUE THREAD

**Issue by FelixMoelder** [romkatv/gitstatus#196]
It looks like the interal version has not been increased for the latest releases.
When building gitstatus 1.4.3 and calling `gitstatusd --version` it still returns `v1.3.1`.

**Comment by romkatv** [comment]
https://github.com/romkatv/gitstatus contains the source code for gitstatusd plus shell bindings for bash and zsh. There have been no changes in the source code of gitstatusd since v1.3.1, so the behavior of `gitstatusd` hasn't changed since v1.3.1, so `gitstatusd --version` prints `v1.3.1`.

The advantage of this scheme is that users who update gitstatus with `git pull` don't have to download/build a new version of gitstatusd if changes in gitstatus are confined to the shell code.

tl;dr: Working as intended.

**Comment by FelixMoelder** [comment]
Thanks for clarifying. This makes sense for the scenario you described, while it stays confusing in terms of distinct release versions.
